frontend/src/Components/LayoutGenerator.tsx

```diff
@@ -145,10 +146,10 @@ const LayoutGenerator: FC = () => {
                 })}
             </Responsive>
         </div >,
-        <div id="filter-tab">
-            <div className={"lineup"} id={"lineup-wrapper"}>
-                {/* <LineUpWrapper hemoglobinDataSet={hemoData} /> */}
-            </div>
+        <div id="filter-tab" style={{ height: "90vh" }}>
+            <FilterBoard />
+            {/* <LineUpWrapper hemoglobinDataSet={hemoData} /> */}
```

Comment:
Let's make an issue to remove lineup if that's our next step

Reply:
I'm actually fine with removing lineup completely in this PR.

I have a local version of the old src, and that has lineup set up in there. So we want to add it back in, I can refer to my local files. 

I'll remove it completely.